<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget_ShowData</class>
 <widget class="QWidget" name="Widget_ShowData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1244</width>
    <height>778</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>120</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget,
QMainWindow{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(80, 80, 90, 255), stop:1 rgba(30, 30, 30, 255));
    font: 12pt &quot;微软雅黑&quot;;
	color: rgb(194, 193, 187);
}
QTableWidget{
	alternate-background-color: rgba(255, 255, 255, 20);
}

/*表头样式*/

QHeaderView::section{
	background-color: rgb(86, 86, 86);
	border:1px solid rgba(220,220,220,100);
	color: rgb(255, 255, 255);

}
/*按钮*/
QPushButton,
QToolButton{
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(68, 68, 75, 255), stop:1 rgba(30, 30, 30, 255));
	border:1px groove rgba(255,255,255,100);

}
/*按钮悬停和按下状态*/
QPushButton:hover,
QPushButton:pressed,
QPushButton:checked,
QPushButton:selected,
QToolButton:hover,
QToolButton:pressed,
QToolButton:checked,
QToolButton:selected{
	color: rgba(255, 150, 0,255);

	background-color: qradialgradient(spread:pad, cx:0.505682, cy:0, radius:0.8, fx:0.505682, fy:0, stop:0 rgba(106, 162, 214, 255), stop:1 rgba(30, 30, 30, 255));
	border:2px groove rgba(70, 140, 195,100);
}
/*滚动条*/
QScrollBar{
	background-color: rgba(255, 255, 255,10);
	border:1px groove  gay;	
	border-color: rgba(255, 255, 255,80);
}
/*水平滚动条*/
QScrollBar:horizontal{
    height:17px;
    margin:0 16 0 16;
}
/*水平滚动条 滑块*/
QScrollBar::handle:horizontal{
	background-color: rgba(255,255,255,95);
    min-width:20px;
    margin:1px 0px 1px 0px;
	border:1px groove  gay;	
	border-color: rgba(255, 255, 255,80);
}
QScrollBar::handle:horizontal:hover{
	background-color:rgba(70, 140, 195,100);	
    min-width:20px;
    margin:1px 0px 1px 0px;
	border:1px groove  gay;	
	border-color: rgba(255, 255, 127,50);
}
/*水平滚动条 微调按钮*/
QScrollBar::add-line:horizontal{
	background-color: rgba(255,255,255,60);
    border:1px groove  gay;	
	border-color: rgba(255, 255, 255,80);
    width:15px;
    subcontrol-position:right;
    subcontrol-origin:margin;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="5">
    <widget class="QPushButton" name="pushButton_5">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(22, 155, 213);</string>
     </property>
     <property name="text">
      <string>最小化</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>      数据库</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QPushButton" name="pushButton_4">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(22, 155, 213);</string>
     </property>
     <property name="text">
      <string>关闭</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="3">
    <widget class="QListWidget" name="listWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pushButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(22, 155, 213);</string>
     </property>
     <property name="text">
      <string>导入</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>2</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="pushButton_dr">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);

</string>
     </property>
     <property name="text">
      <string>导入到环境</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="pushButton_up">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
</string>
     </property>
     <property name="text">
      <string>修改数据库信息</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QPushButton" name="pushButton_delete">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
</string>
     </property>
     <property name="text">
      <string>删除数据库</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="5">
    <widget class="QTableWidget" name="tableWidget">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <column>
      <property name="text">
       <string>帐号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>帐号类型</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>柜台类型</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>帐号属性（标签）</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>所属数据库</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>操作</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="3" colspan="2">
    <widget class="QLineEdit" name="lineEdit">
     <property name="styleSheet">
      <string notr="true">background-color: rgba(120, 120, 120,120);</string>
     </property>
    </widget>
   </item>
   <item row="3" column="6">
    <widget class="QPushButton" name="pushButton_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(22, 155, 213);</string>
     </property>
     <property name="text">
      <string>数据合并</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QPushButton" name="pushButton_2">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(22, 155, 213);</string>
     </property>
     <property name="text">
      <string>查找</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
